### PR TITLE
Photometry Follow Up

### PR DESCRIPTION
**Pull request by pauladkisson**
Implement feedback from Lerner Lab on Photometry Conversion

TODOs
- [x] Add support for partially corrupted photometry files
- [x] Fix mismatched epoc names
- [x] Stitch multifile photometry sessions
- [x] Ignore incorrect med files
- [x] Add missing behavior session
- [x] Add support for Fi1r-only sessions
- [x] Add missing RR20 data

### CODE REVIEW

File: src/lerner_lab_to_nwb/seiler_2024/seiler_2024_convert_session.py
Comment (alessandratrapani) [inline src/lerner_lab_to_nwb/seiler_2024/seiler_2024_convert_session.py:1]: Is there a reason for keeping all of this commented lines?

**Review by alessandratrapani — COMMENTED** [review]
OK!

File: src/lerner_lab_to_nwb/seiler_2024/seiler_2024_convert_session.py
Comment (pauladkisson) [reply src/lerner_lab_to_nwb/seiler_2024/seiler_2024_convert_session.py:1]: Yes, this is the list of all the example sessions that I use to informally test various branches and edge cases in the conversion.  I often comment/uncomment them depending on which aspects of the conversion I am working on atm. Probably shouldn't be commented out in main tho...

**Comment by pauladkisson** [comment]
> OK!

Is this good to go? If so pls approve. Thanks!

**Review by alessandratrapani — APPROVED** [review]

**Pull request merged**